lib/princely/rails.rb

```diff
@@ -7,6 +7,7 @@
 if defined?(Rails)
   if Rails::VERSION::MAJOR >= 5
     ActionController::Base.send(:prepend, Princely::PdfHelper)
+    ActionController::Base.send(:include, Princely::AssetSupport)
```

Comment:
That is no different than the [lines that follow the if block](https://github.com/mbleigh/princely/blob/38786246d7263c5cb567f748acc65780b16260d1/lib/princely/rails.rb#L13-L15).